indico/migrations/versions/201704241604_bc4e7682e7df_fix_regform_old_accommodation_fields.py

```diff
@@ -0,0 +1,58 @@
+"""Fix regform old accommodation fields
+
+Revision ID: bc4e7682e7df
+Revises: 2963fba57558
+Create Date: 2017-04-24 16:04:07.476843
+"""
+
+import json
+import uuid
+import itertools
+from operator import attrgetter
+
+from alembic import context, op
+
+# revision identifiers, used by Alembic.
+revision = 'bc4e7682e7df'
+down_revision = '2963fba57558'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    if context.is_offline_mode():
+        raise Exception('This upgrade is only possible in online mode')
+    conn = op.get_bind()
+
+    field_data_query = conn.execute("""
+        SELECT field_id, ffd.id, fi.is_required, ffd.versioned_data, fi.data
```

Comment:
I'd always qualify field names when more than one table is involed. i.e. `ffd.field_id` (or wherever it comes from) instead of just `field_id`